packages/turf-voronoi/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@turf/voronoi",
+  "version": "0.1.0",
+  "description": "turf module to generate Voronoi polygons from a set of points and a bounding box",
```

Comment:
We are using JSDocs for the description of the modules, since no one will be maintaining the `package.json` we just define this description as very generic.

```json
"description": "turf voronoi module",
```